### ISSUE TITLE
Soho.theme.personalizationColors().default color is azure

### ISSUE THREAD

**Issue by anhallbe** [infor-design/enterprise#7167]
**Describe the bug**
When calling `Soho.theme.personalizationColors()` (Or `SohoPersonalizeDirective.personalizationColors()`), the `default` color value is `#0072ED` (azure).

Since 4.80, I assume this should be white/alabaster (`#ffffff`)?

**To Reproduce**

1. In an application that is running `ids-enterprise@4.80`, execute `console.log(Soho.theme.personalizationColors().default.value)`
2. See `#0072ED`

**Expected behavior**
I *think* the default color should be `#ffffff` or whatever color alabaster is.

**Version**

- ids-enterprise: 4.80
- ids-enterprise-ng: 15.0.1

**Additional context**
For some reason (need investigation), our app primarily uses the "azure" color even if "default is set.

https://jira.infor.com/browse/SCRMPORTALV2-5669

We are using the output from `SohoPersonalizeDirective.personalizationColors()` select and persist the personalisation color selected by the user.

In essence, this code (which worked before, and still works for all colors **except the default**):
```js
// This makes things azure:
this.personalizeDirective.colors = this.personalizeDirective.personalizationColors().default.value;

// This also makes things azure:
this.personalizeDirective.colors = this.personalizeDirective.personalizationColors().azure.value;
```

**Comment by anhallbe** [comment]
So I managed to work around the problem similar to how it is done in the `ids-enterprise-ng` samples. Something like:

```ts
setColorById(id: "default" | "azure" | "amber" /* ... */) {
  const colors = this.personalizeDirective.personalizationColors();
  const color = colors.find(c => c.id === id);
  
  if (color.id === "default") {
    this.personalizeDirective.colors = ""; // This appears to work
  } else {
  this.personalizeDirective.colors = color.value;
  }
}
```

There are *a lot* of issues related to the new default color though. Not sure if they are caused by this workaround. For instance, `.header.is-personalizable` is now correctly white, but `.is-personalizable .personalize-subheader` is blue (#0054b1) with only some text elements getting the correct color.

**Comment by tmcconechy** [comment]
We will have to raise each issue / or group of issues related to things not working with `is-personalizable`. But we do have a handful already in this sprint that we know about https://github.com/orgs/infor-design/projects/139

Will keep this issue related to the personalization not working. That fix looks ok to me...

**Comment by tmcconechy** [comment]
FYI issue will be patched in 4.80.2 / NG 14.8.2 ~ running shortly
